google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/AddCloudLibrariesAction.java

```diff
@@ -68,52 +63,53 @@ public void actionPerformed(AnActionEvent e) {
     }
   }
 
-  /** The wizard for the "Add Cloud Libraries" menu action. */
-  // TODO(eshaul) this does not need to be a wizard and can be converted to a simple dialog.
-  private static final class AddCloudLibrariesWizard extends AbstractWizard<Step> {
+  /** The dialog for the "Add Cloud Libraries" menu action. */
+  private static final class AddCloudLibrariesDialog extends DialogWrapper {
 
     private final Project project;
-    private final SelectClientLibrariesStep selectClientLibrariesStep;
-
-    AddCloudLibrariesWizard(Project project) {
-      super(GctBundle.message("cloud.libraries.dialog.title"), project);
+    private final GoogleCloudApiSelectorPanel cloudApiSelectorPanel;
 
-      this.project = project;
+    AddCloudLibrariesDialog(Project project) {
```

Comment:
This is great, so much easier to read! Could we also consider making `AddCloudLibrariesDialog` its own top-level class, since it does not have any real dependencies on `AddCloudLibrariesAction` and it's pretty large?

Reply:
yes. good idea.